ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_linalg.py

```diff
@@ -657,3 +657,37 @@ def test_paddle_cond(*, dtype_and_x, p, on_device, fn_tree, frontend, test_flags
         atol=1e-5,
         p=p,
     )
+
+# dot
+@handle_frontend_test(
+    fn_tree="paddle.tensor.linalg.dot", 
+    dtype_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        shape=(1,),
+        min_value=-10,
+        max_value=10,
```

Comment:
I guess if we used the default values of the strategy would be better

Reply:
add_frontend_checklist

Reply:
Paddle has a habit of turning really large numbers into (inf) or really small numbers into (-inf) when other frameworks don't, and it causes the internal tests to fail. And that's also what others did in their tests.

Reply:
Unit test should pass on every valid case on paddle, if paddle behaves in a certain way, we should expect the frontend function to behave the same.

Reply:
Aside from the issue above (which I couldn't replicate today to be honest), the problem is that the error range of the absolute difference is around atol = 1e-06 , and the acceptable range is atol=1e-08. So I am not sure if I can get the error lower but I am going to try to make it work :) if I couldn't I will push the changes done to vectorizing the function. 

Another note: I see that most functions implemented in linalg.py file either use a range of values like I did before or they lower the acceptable error. I will try the second approach if I run out of options and leave it to your evaluation :).